Makefile

```diff
@@ -156,6 +156,7 @@ release-prep:
 	@cp -r ./plugins release_deps/
 	@cp config/example.yaml release_deps/config.yaml
 	@cp config/logging.yaml release_deps/logging.yaml
+	@jq ".files[] | select(.service != null)" windows/wix.json >> release_deps/windows_service.json
```

Comment:
@BinaryFissionGames could you explain what this line of text is doing?

Reply:
`.files[]` selects the items in the array "files" in `windows/wix.json`, then it's piped into a "select" statement, which filters out all items in the `files` array that doesn't have a service definition. So we end up with a file that looks like the one in this PR description.